packages/garbo/src/tasks/cockroachPrep.ts

```diff
@@ -97,8 +145,18 @@ function chooseCrew(): void {
   runChoice(bestChoice);
 }
 
+const funPointValue = garboValue($item`PirateRealm guest pass`) / 600;
+const carnPlantValue =
+  get("valueOfAdventure") / (20 + get("_carnivorousPottedPlantWins"));
+const funPointBonuses = new Map<Item, number>([
+  [$item`Red Roger's red left foot`, funPointValue],
+  [$item`PirateRealm party hat`, funPointValue],
+  [$item`carnivorous potted plant`, carnPlantValue],
+]);
```

Comment:
```suggestion
function outfitBonuses() {
  const funPointValue = garboValue($item`PirateRealm guest pass`) / 600;
  return new Map([
    [$item`carnivorous potted plant`,   get("valueOfAdventure") / (20 + get("_carnivorousPottedPlantWins"))],
    [$item`Red Roger's red left foot`, funPointValue],
    [$item`PirateRealm party hat`, funPointValue],
  ]);
}
```

we don't want mallsearches in top level scope (because they'll lead to mallsearches before we do `mallPrices("allitems")` and we don't want carn plant in top level scope (because we don't want it to stay fixed at the initial value